content.md

```diff
@@ -741,17 +733,11 @@ effect after you have unlocked your cards. You may choose the order in
 which these effects take place. Along with effects generated by cards,
 there are other effects that are resolved during the unlock phase:
 
-If you have the Edge, you may gain 1 pool from the blood bank.
-
-> **ADVANCED RULES**\
-> \
-> Some cards have a burn option icon.\
-> \
-> This icon means that a Methuselah who does not control a minion who
-> meets the requirements of this card or is not a legal target, may discard it during ANY
-> Methuselah's unlock phase and replace it.\
-> \
-> Each Methuselah is limited to one such discard each unlock phase.\
+- If you have the **Edge**, you may gain 1 pool from the blood bank.\
+- If a **card you control is contested**, you may decide to continue the 
+contest.\
+- If a **title of a vampire you control is contested**, you may decide to 
+continue the contest.
```

Comment:
Or 
```suggestion
- If you have an ongoing **contest**, you must decide whether to pay for it or to yield. (see [contest](#useful-link-here))
```

Reply:
Would a link really be needed if the explanation comes right after?